config/base/vsa-signing-key-secret.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright The Conforma Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vsa-signing-key
+  namespace: default
+type: Opaque
+data:
+  # Base64 encoded private key for VSA signing
+  # To generate: openssl genpkey -algorithm ed25519 -out cosign.key
+  # To encode: base64 -w 0 cosign.key
+  cosign.key: |
+    LS0tLS1CRUdJTiBFTkNSWVBURUQgU0lHU1RPUkUgUFJJVkFURSBLRVktLS0tLQpleUpyWkdZaU9uc2libUZ0WlNJNkluTmpjbmx3ZENJc0luQmhjbUZ0Y3lJNmV5Sk9Jam8yTlRVek5pd2ljaUk2Ck9Dd2ljQ0k2TVgwc0luTmhiSFFpT2lJNGQxcDBPWGxhZWtvdlpsUktSemxNTlZwbmVGTnRNMWxUTDNaRFNHeFUKVmtST01URTNOVlphV0U5QlBTSjlMQ0pqYVhCb1pYSWlPbnNpYm1GdFpTSTZJbTVoWTJ3dmMyVmpjbVYwWW05NApJaXdpYm05dVkyVWlPaUpRVjBadGJYQjBhbXhUTHpOdGVWWmpUWEZRTldobE1IZENkMWN5VDB4MFR5SjlMQ0pqCmFYQm9aWEowWlhoMElqb2lTVWc1Wm5ORlQyaGFia1JvZFROTVlXdE5lVGhzTnpsTFRYTnVXV1ZLU1ZOd01uWlMKWnk5VE9FUklXV2QyWWtkcmJqUm1OVzluVmxBelIxVlZMM0l6Y25oS1drcERjSEF6VkdGUFRtMTROR056TmxaRApjRTl4VW5WM09GZHlUV2xFYlc5b2JrcFlMMHhoVTBWQ09GUnJaVTR3VmxwUFQzaFhUMmRDSzJWUWF6UmxaVWRwCmNUVmtjRFIwVG1kUmRXWm1WSEEwY0ZCV2JraEdXR05GWm5wblFqZFNPRmcyYTNSV1RtZFJSV2hwU0ZCb1oyTkoKYmpONWJERlRVVmg0Tm1OaEwwVXJWMjE0YVROYWFXVlBVbWM5UFNKOQotLS0tLUVORCBFTkNSWVBURUQgU0lHU1RPUkUgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
1. Don't we need a public for this also?
2. Maybe not a blocker, but we could generate this on the fly rather than git add a real hard-coded key.